packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -176,17 +176,26 @@ export default class PciInstancesAddController {
   }
 
   getFilteredRegions() {
+    this.productRegionsAllowed = [];
```

Comment:
Same as https://github.com/ovh/manager/pull/11302#discussion_r1541329452, productRegionsAllowed is not used outside this function and can be stored in a local variable